belgif-rest-problem-spring/src/main/java/io/github/belgif/rest/problem/spring/OpenApiRequestViolationProblemAdvice.java

```diff
@@ -0,0 +1,180 @@
+package io.github.belgif.rest.problem.spring;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import com.atlassian.oai.validator.report.ValidationReport;
+import com.atlassian.oai.validator.springmvc.InvalidRequestException;
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.github.belgif.rest.problem.BadRequestProblem;
+import io.github.belgif.rest.problem.ResourceNotFoundProblem;
+import io.github.belgif.rest.problem.api.InEnum;
+import io.github.belgif.rest.problem.api.InputValidationIssue;
+import io.github.belgif.rest.problem.api.InputValidationIssues;
+import io.github.belgif.rest.problem.api.Problem;
+import io.swagger.v3.oas.models.parameters.Parameter;
+
+/**
+ * This class is only active when Spring-Boot applications implemented the Atlassian swagger-request-validator.
+ * Maps the InvalidRequestException thrown by the swagger-request-validator to belgif problems.
+ */
+@RestControllerAdvice
+@ConditionalOnWebApplication
+@ConditionalOnClass(InvalidRequestException.class)
+@Order(1)
+// @Order(1) to take precedence over io.github.belgif.rest.problem.spring.ProblemExceptionHandler
+public class OpenApiRequestViolationProblemAdvice {
```

Comment:
The naming pattern is different from the existing exception handlers:
* ProblemExceptionHandler
* RoutingExceptionsHandler
* BeanValidationExceptionsHandler
* NoResourceFoundExceptionHandler

(well... they are already a bit inconsistent by themselves with ExceptionHandler vs ExceptionsHandler).